dali/kernels/signal/fft/fft_postprocess_test.cu

```diff
@@ -106,7 +106,8 @@ class FFTPostprocessTest<FFTPostprocessArgs<Out, In, Convert>> : public ::testin
     auto cpu_in = in.cpu();
     UniformRandomFill(cpu_in, rng, -1, 1);
 
-    ToFreqMajorSpectrum<Out, In, Convert> tr;
+    Convert convert;
+    ToFreqMajorSpectrum<Out, In, Convert> tr(convert);
```

Comment:
What's the point of this? The default constructor of ToFreqMajorSpectrum would use a default-constructed Convert. Now this function tests a different constructor. Why?

Reply:
As we're already have the variable of that type, I just thought to move it and make the coverity happy. The power_dB is not an empty object, still it should be default-initialized, shouldn't it?

Reply:
I can revert it if you want.